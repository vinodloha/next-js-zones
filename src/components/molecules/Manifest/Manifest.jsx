import React from 'react'
import HeadLink from '../../atoms/HeadLink'

const Manifest = () => {
  return(
    <HeadLink
      rel="manifest"
      href="/site.webmanifest"
    />
  )
}

export default Manifest
