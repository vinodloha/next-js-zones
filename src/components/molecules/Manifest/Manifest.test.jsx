import React from 'react'
import TestRenderer from 'react-test-renderer'
import Manifest from '.'

describe('Manifest', () => {
  it('creates a manifest <link> element suitable for the document head', () => {
    const result = TestRenderer.create(
      <Manifest />
    ).toJSON()

    expect(result).toMatchSnapshot()
  })
})
