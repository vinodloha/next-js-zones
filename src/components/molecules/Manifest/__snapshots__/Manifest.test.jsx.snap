// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Manifest creates a manifest <link> element suitable for the document head 1`] = `
<link
  href="/site.webmanifest"
  rel="manifest"
/>
`;
